frontend/src/container/MetricsApplication/constant.ts

```diff
@@ -3,7 +3,12 @@ export const legend = {
 };
 
 export const QUERYNAME_AND_EXPRESSION = ['A', 'B', 'C'];
-export const LETENCY_LEGENDS_AGGREGATEOPERATOR = ['p50', 'p90', 'p99'];
+export const LATENCY_LEGENDS_AGGREGATEOPERATOR = ['p50', 'p90', 'p99'];
+export const LATENCY_LEGENDS_AGGREGATEOPERATOR_SPAN_METRICS = [
+	'hist_quantile_50',
+	'hist_quantile_90',
+	'hist_quantile_99',
+];
```

Comment:
There is no need to change the legend here.

Reply:
I don't think I really I fully understand this.